Neos.Eel/Classes/Utility.php

```diff
@@ -36,12 +36,31 @@ public static function getDefaultContextVariables(array $configuration)
                 }
                 $currentPathBase = & $currentPathBase[$pathName];
             }
-            $currentPathBase = new $objectType();
+            $currentPathBase = self::createContextObjectFromConfiguration($objectConfiguration);
         }
 
         return $defaultContextVariables;
     }
 
+    /**
+     * @param $objectConfiguration
+     * @return callable|object
+     */
+    public static function createContextObjectFromConfiguration($objectConfiguration)
```

Comment:
```suggestion
    private static function createContextObjectFromConfiguration(array $objectConfiguration)
```
can be private and the param is an array, right